src/client/components/dashbord-items/dashboard-items.component.js

```diff
@@ -6,13 +6,17 @@ const DashbordItems = ({ items }) => {
   return (
     <ul className="list">
       {items.map((item) => (
-        <li key={item.id}>{item.value}</li>
+        <li key={item.id}>
+          <a href={`/chart-detail-page/${item.value.toLowerCase()}`}>
```

Comment:
Probably want to user the `<Link>` component from React Router :)

Reply:
I removed those changes since dashboard items (in sidebar menu) should be visible only on detail chart page and that is not part of this PR